tests/rest/client/test_rooms.py

```diff
@@ -1116,6 +1116,320 @@ def test_rooms_messages_sent(self) -> None:
         self.assertEqual(200, channel.code, msg=channel.result["body"])
 
 
+class RoomPowerLevelOverridesTestCase(RoomBase):
+    """Tests that the power levels can be overridden with server config."""
+
+    user_id = "@sid1:red"
+
+    servlets = [
+        admin.register_servlets,
+        room.register_servlets,
+        login.register_servlets,
+    ]
+
+    def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
+        self.admin_user_id = self.register_user("admin", "pass")
+        self.admin_access_token = self.login("admin", "pass")
+
+    def power_levels(self, room_id: str) -> Dict[str, Any]:
+        return self.helper.get_state(
+            room_id, "m.room.power_levels", self.admin_access_token
+        )
+
+    def test_default_power_levels_without_override(self) -> None:
```

Comment:
Hmm, this test will fail every time we change the default power levels. As such, it doesn't seem very future-proof.

Likewise in the tests below. Perhaps in those tests, one should only check that the appropriate fields were added/changed?

Reply:
Fixed in ae9421caa